ethereum/statetransition/src/test/java/tech/pegasys/teku/statetransition/util/BlockBlobSidecarsTrackersPoolImplTest.java

```diff
@@ -1296,6 +1293,66 @@ void stats_onNewBlock() {
     assertStats("block", "gossip", 2);
   }
 
+  // TODO: tests on DataColumnSidecarELRecoveryManagerImpl
```

Comment:
Are the tests there? Can we remove the TODO?

Reply:
Yeah, enough tests for this step. Removed